### ISSUE TITLE
Marketing information stores null values

### ISSUE THREAD

**Issue by bavarianbytes** [in2code-de/powermail#879]
Opened the website in an incognito tab, visited a few pages, submitted a powermail form.

The pagefunnel consistents out of empty entries:

<img width="537" alt="Bildschirm­foto 2023-05-19 um 15 57 18" src="https://github.com/einpraegsam/powermail/assets/17915115/6622f7c2-aab6-4715-8bad-b1dd61432e3c">

The information is already not present in the FE user session:

`a:1:{s:9:"powermail";a:1:{s:19:"powermail_marketing";a:7:{s:13:"refererDomain";s:0:"";s:7:"referer";s:0:"";s:7:"country";s:0:"";s:12:"mobileDevice";b:0;s:16:"frontendLanguage";i:0;s:15:"browserLanguage";s:2:"de";s:10:"pageFunnel";a:3:{i:0;i:0;i:1;i:0;i:2;i:0;}}}}`

Seems like all the variables like `$pid` are empty in the `FormController.php` action, here:

```
    public function marketingAction(
        string $referer = '',
        int $language = 0,
        int $pid = 0,
        bool $mobileDevice = false
    ): ResponseInterface {
        SessionUtility::storeMarketingInformation($referer, $language, $pid, $mobileDevice, $this->settings);
        $response = $this->responseFactory->createResponse()
            ->withHeader('Content-Type', 'application/json; charset=utf-8');
        $response->getBody()->write(json_encode([]));
        return $response;
    }
```

TYPO3 11.5.27
powermail 10.7.2
PHP 8.2

**Comment by bavarianbytes** [comment]
It's a bug in the `Marketing.js` file.
I could fix it like this: https://stackoverflow.com/questions/70005727/typo3-cms-version-10-4-with-powermail-8-4-marketing-information/76728326#76728326

Maybe you wanna change it in the extension code too. Sorry, right now i have no time for a fix via a pull request.

**Comment by MrGecco** [comment]
@mschwemer sorry to bother you. any update on this bug? thank you very much in advance

**Comment by mschwemer** [comment]
Solved with 5e06c9b18130625cc2a147e923f086d3c4c164ba
